bigtable/google/cloud/bigtable/table.py

```diff
@@ -317,7 +319,7 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         return PartialRowsData(response_iterator)
 
     def yield_rows(self, start_key=None, end_key=None, limit=None,
-                   filter_=None):
+                   filter_=None, retry=False):
```

Comment:
Please remove the option to not retry.  There is no case where a user doesn't want to retry.

Reply:
The parameter should remain so that users can adjust the retry options, but it should probably have a sane default.

Reply:
Retry should be on by default. This is a streaming response, and no user wants to re-retrieve what they already read. 

If we do need a parameter, then we need to have some sort of ReadRetyOptions class rather than a boolean.  Things like deadlines between streaming responses can be added to that ReadRetyOptions object.

Reply:
Please do not introduce a new class.